templates/base.html

```diff
@@ -8,17 +8,108 @@
+                                                team that reveals all their words first wins.
+                                            </p>
+                                            <h5>The Spymaster</h5>
+                                            <p>
+                                                Each turn starts with the Spymaster giving a clue to their operative. A
+                                                clue consists of a word and the number of words in the team color that
+                                                it relates to.
+                                            </p>
+                                            <h5>The Operative</h5>
+                                            <p>
+                                                The Operatives guess the team's words based on the Spymaster's clue. If
+                                                the operator guesses a word correctly, he may guess again. The goal is
+                                                to guess as many words as the spymaster has indicated. He can also guess
+                                                unsolved clues from previous rounds until the maximum number of guesses
+                                                (clue number + 1) is reached.
+                                                The turn can end in different ways. If a word is revealed on a neutral
+                                                card or in the opponent's color, the Operative may not give another
+                                                Guess. The turn can also be ended manually or by reaching the maximum
+                                                number of guesses.
+                                            </p>
+                                            <h5>Winning and loosing</h5>
+                                            <p>
+                                                A team wins when all the words of their own color have been guessed.
+                                                They lose the game by revealing the black card.
+                                            </p>
+                                            </p>
+                                        </div>
+                                        <div class="modal-footer">
+                                            <button type="button" class="btn btn-secondary"
+                                                    data-bs-dismiss="modal">Close</button>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </li>
+                    </ul>
+                    <div class="d-flex">
+                        <input id="idCodeInput" class="form-control me-1" type="text" placeholder="Enter Code!">
```

Comment:
I suggest changing the behaviour of the input and the button to a html post form. This can ensure backend validation. Additionally, as the room code will consist of 6 characters (current state) a `minlength="6"` and `maxlength="6"` would be an idea to implement pre-validation. Also, adding a `required oninvalid="this.setCustomValidity("message")"` could be used to give a custom message.